backend/otel/otel.go

```diff
@@ -600,17 +600,36 @@ func (o *Handler) submitProjectLogs(ctx context.Context, projectLogs map[string]
 }
 
 func (o *Handler) submitTraceSpans(ctx context.Context, traceRows map[string][]*clickhouse.TraceRow) error {
+	markBackendSetupProjectIds := map[uint32]struct{}{}
+	projectIds := map[uint32]struct{}{}
+	for _, traceRows := range traceRows {
+		for _, traceRow := range traceRows {
+			// Skip traces with a `http.method` attribute as likely autoinstrumented frontend traces
+			if _, found := traceRow.TraceAttributes["http.method"]; !found {
+				markBackendSetupProjectIds[traceRow.ProjectId] = struct{}{}
+			}
```

Comment:
could convert this to using the `highlight.type==highlight.internal` newer functionality

Reply:
:+1: updated to use the same logic as `trace_count_daily_mv`